ports/ceres/portfile.cmake

```
@@ -40,6 +40,8 @@ vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
         ${FEATURE_OPTIONS}
+        -DGFLAGS=OFF
+        -DMINIGLOG=ON
```

```suggestion
        -DMINIGLOG=OFF
```
It should be `OFF` to enable to `find_package(glog)` instead of `miniglog` inside port `ceres`. https://github.com/ceres-solver/ceres-solver/blob/344929647a7ab3df6b60bce803af6db4c1a4c186/CMakeLists.txt#L400
Otherwise, adding `glog` in dependencies will not look for `glog`. :)